src/api/uniqueSdk/NFTController.ts

```diff
@@ -59,12 +59,26 @@ export class UniqueSDKNFTController implements UniqueController {
     return await this.sdk?.tokens.getBundle({ collectionId, tokenId }) || null;
   }
 
-  async unnest(nested: { collectionId: number, tokenId: number }, parent: { collectionId: number, tokenId: number }, options: TransactionOptions): Promise<void> {
-    if (!options.signer) return;
+  async unnest(address: string, nested: { collectionId: number, tokenId: number }, parent: { collectionId: number, tokenId: number }, options: TransactionOptions): Promise<void> {
```

Comment:
unnest is always a transfer to self, no need to pass "address" as param here. Same for the rest.For unnest you need only collectionId and tokenId.